SimTracker/SiPhase2Digitizer/plugins/Pixel3DDigitizerAlgorithm.h

```diff
@@ -0,0 +1,69 @@
+#ifndef _SimTracker_SiPhase2Digitizer_Pixel3DDigitizerAlgorithm_h
```

Comment:
Typically we recommend plugins should not have headers, since their code is not supposed to be shared. Is there any reason this plugin needs a separate header file? If not, the class definition should move into the .cc file.

Reply:
Mainly because of consistency, as I found that all the other plugins in the package (which are intended not to be shared as well) are keeping this structure.